src/performanceconfig.h

```diff
@@ -113,6 +131,17 @@ class CPerformanceConfig	// Performance configuration
 	unsigned m_nPortamentoMode[CConfig::ToneGenerators];
 	unsigned m_nPortamentoGlissando[CConfig::ToneGenerators];
 	unsigned m_nPortamentoTime[CConfig::ToneGenerators];
+	std::string m_nVoiceDataTxt[CConfig::ToneGenerators]; 
+	
+	unsigned nLastPerformance;  
+	unsigned nLastFileIndex;
+	unsigned nActualPerformance = 0;  
+	unsigned nMenuSelectedPerformance = 0;
+	std::string m_nPerformanceFileName[40];
```

Comment:
Does this limit the number of performances to 40? https://github.com/probonopd/MiniDexed/issues/379